pkg/services/ngalert/ngalert.go

```diff
@@ -185,6 +185,9 @@ func (ng *AlertNG) init() error {
 
 		case remoteSecondary:
 			ng.Log.Debug("Starting Grafana with remote secondary mode enabled")
+			m := ng.Metrics.GetRemoteAlertmanagerMetrics()
+			m.Info.WithLabelValues("remote_secondary").Set(1)
```

Comment:
should these be constants? Would be nice to convert `remoteSecondary` to its label counter-part.